package.json

```diff
@@ -184,6 +184,7 @@
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
     "styled-components": "^5.3.6",
+    "uuid": "^9.0.0",
```

Comment:
Already included as transient dependency, now included as a direct one

<img width="417" alt="image" src="https://user-images.githubusercontent.com/17035424/227188715-8cbc3d08-4c8e-48b4-9abe-f317da5ca45b.png">

We could use react's `useId()` hook for that with an HOC / wrapper component but IMO that introduces more complexity than is necessary here

Reply:
Another way is to use e.g `Math.random()` or `Date.now()` if we don't want to add a new dep, which would achieve the same effect

Reply:
90b3a562fc6b75a6cef765ed682feb3753300883